### ISSUE TITLE
lang cookie misses "Secure" flag

### ISSUE THREAD

**Issue by fkooman** [OpenConext/OpenConext-engineblock#831]
> Cookies with SameSite=None must also specify Secure, meaning they require a secure context.

```
lang=en; expires=Mon, 13-Apr-2020 09:35:36 GMT; Max-Age=5180400; path=/; domain=.surfconext.nl; SameSite=None
```

**Comment by thijskh** [comment]
Since SameSite is set in the LB I would also fix this there at this point

**Comment by MKodde** [comment]
@fkooman It took some figuring out, but it turns out EngineBlock allows manipulation the lang cookie settings. In the application INI file you can use the following options:

```ini
cookie.lang.domain
cookie.lang.expiry
cookie.lang.http_only ; (defaults to false)
cookie.lang.secure ; (defaults to false)
```

So updating your application/engineblock .ini file should remedy your problem.

**Comment by fkooman** [comment]
> So updating your application/engineblock .ini file should remedy your problem.

Thanks! I'm sorry, I hope I didn't cause too much inconvenience! I was just cruising by... I'm not actually running EB, was just curious how you guys handled `SameSite` and noticed this issue...

**Comment by MKodde** [comment]
Not a problem! We might change the default to comply with the best practices. Or at least document the possible configuration options.

**Comment by tvdijen** [comment]
Maybe it's just me, but a load balancer is not the place to correct all sorts of stuff that the application lacks to do properly..

**Comment by thijskh** [comment]
Sure, but we had to be pragmatic and could not update 16 applications all at once, so we opted to do it in the LB for all apps at once.

**Comment by thijskh** [comment]
Fixed in EB 6.2.
